source/template.yaml

```diff
@@ -54,17 +58,45 @@ Globals:
       environment: !Ref environment
 
 Resources:
+  # add all functions that need gitsha alias to this list
```

Comment:
this is a reordering (since there was a group of 3 functions together, and a fourth up top), not an actual change